aten/src/ATen/DLConvertor.cpp

```diff
@@ -40,6 +40,9 @@ static DLDataType getDLDataType(const Tensor& t) {
     case ScalarType::Bool:
       dtype.code = DLDataTypeCode::kDLUInt;
       break;
+    case ScalarType::QInt8:
```

Comment:
I'd just fail here as DLPack doesn't support quantization at all.

Reply:
that sounds right -- why was this resolved?